lib/poliplot.m

```diff
@@ -0,0 +1,68 @@
+fsamp=125*10^6; %sampling frequency of the signal
```

Comment:
PLease add a short description at the beginning of the .m file, explaining that this is a `main `code, what's the purpose, etc...